data/FilterList.json

```diff
@@ -8969,8 +8919,7 @@
     "licenseId": 2,
     "name": "Magno's Adblock List",
     "syntaxId": 3,
-    "viewUrl": "https://raw.githubusercontent.com/magnobiet/adblock-plus-list/gh-pages/list.txt",
-    "viewUrlMirror1": "https://adp.magnobiet.com/list.txt"
+    "viewUrl": "https://adp.magnobiet.com/list.txt",
```

Comment:
maybe this extra trailing comma? I'm not sure. something is goofy with the Travis build which used to help isolate errors.